cli/types/cypress.d.ts

```diff
@@ -2993,6 +2993,7 @@ declare namespace Cypress {
     // Internal or Unlisted at server/lib/config_options
     namespace: string
     projectRoot: string
+    repoRoot: Readonly<string | null>
```

Comment:
Maybe this doesn't make sense after all - I suppose if we *really* want it to be readonly, we will need something at runtime too.

Since this is undocumented, maybe it's fine to not mark as `Readonly`. We have no commitment to maintaining an undocumented property.

Could go either way on this, up to you, just thinking out loud - not a blocking change request.